livre/eq/Channel.cpp

```diff
@@ -291,6 +292,7 @@ struct Channel::Impl
         renderPipeline.render( pipe->getFrameData()->getVRParameters(),
                                _frameInfo,
                                {{ _drawRange.start, _drawRange.end }},
+                               pipe->getFrameData()->getVolumeSettings().getDataSourceRange(),
```

Comment:
:) I was afraid somebody will ask for it ( You are not the first, but you are right, I am postponing the in-evidable )